modules/programs/fzf.nix

```
@@ -88,6 +88,25 @@ in {
       '';
     };
 
+    useTmuxInShellIntegration = mkOption {
+      default = false;
+      type = types.bool;
+      description = ''
+        Whether to set FZF_TMUX to 1 which causes shell integration to use fzf-tmux.
+      '';
+    };
+
+    tmuxOptionsInShellIntegration = mkOption {
+      type = types.listOf types.str;
+      default = [ ];
+      example = [ "-d 40%" ];
+      description = ''
+        If useTmuxInShellIntegration is set to true,
+        shell integration will use these options for fzf-tmux.
+        See fzf-tmux --help for available options.
+      '';
+    };
+
```

Could these options be nested under a `tmux` setting?

```suggestion
tmux = {
    enableShellIntegration = mkEnableOption ''
        setting FZF_TMUX to 1 which causes shell integration to use fzf-tmux
      '';
    };

    shellIntegrationOptions = mkOption {
      type = types.listOf types.str;
      default = [ ];
      example = [ "-d 40%" ];
      description = ''
        If <options>programs.fzf.tmux.enableShellIntegration<options> is set to true,
        shell integration will use these options for fzf-tmux.
        See fzf-tmux --help for available options.
      '';
    };
}
```